test/e2e/bundle_e2e_test.go

```diff
@@ -47,6 +118,30 @@ var _ = Describe("Installing bundles with new object types", func() {
 				imageName  = "quay.io/olmtest/single-bundle-index:pdb"
 			)
 
+			// create VPA CRD on cluster
+			data, err := yaml.YAMLToJSON([]byte(vpaCRDManifest))
+			Expect(err).ToNot(HaveOccurred(), "could not convert vpa crd to json")
+
+			crd := &apiextensionsv1beta1.CustomResourceDefinition{}
+			err = json.Unmarshal(data, crd)
+			Expect(err).ToNot(HaveOccurred(), "could not convert vpa crd to unstructured")
+
+			_, err = kubeClient.ApiextensionsInterface().ApiextensionsV1beta1().CustomResourceDefinitions().Create(context.TODO(), crd, metav1.CreateOptions{})
```

Comment:
Can we use the [controller-runtime client](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/test/e2e/ctx/ctx.go#L78) instead?